src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -22,13 +23,16 @@ public static class SentryNative
     /// </summary>
     /// <param name="options">The Sentry Unity options to use.</param>
     /// <param name="sentryUnityInfo">Infos about the current Unity environment</param>
-    public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentryUnityInfo)
+    public static void Configure(SentryUnityOptions options) =>
+        Configure(options, SentryPlatformServices.UnityInfo);
+
+    internal static void Configure(SentryUnityOptions options, ISentryUnityInfo? sentryUnityInfo)
     {
         Logger = options.DiagnosticLogger;
 
         Logger?.LogInfo("Attempting to configure native support via the Native SDK");
 
-        if (!sentryUnityInfo.IsNativeSupportEnabled(options, ApplicationAdapter.Instance.Platform))
+        if (!sentryUnityInfo?.IsNativeSupportEnabled(options, ApplicationAdapter.Instance.Platform) ?? false)
```

Comment:
`sentryUnityInfo` being null is a bigger problem, no?

Shoujld we split this code so we can log specifically `sentryUnityInfo` being null as an error. And the native support being off can stay as debug, as-is?

Reply:
Reworked the PlatformServices.UnityInfo handling by pulling this into the options.